File: ASP.Net/MyBookShelfBackend/Models/Book.cs
namespace MyBookShelfBackend.Models
{
    public class Book
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Author { get; set; }
        public string Description { get; set; }
        public string[] Comments { get; set; }
        public string[] Categories { get; set; }
        public string Rating { get; set; }
        public string Format { get; set; }
        public string PageNumber { get; set; }
        public string Alphabet {  get; set; }
        public string Cover { get; set; }
        public DateOnly ReleaseDate { get; set; }
        public bool Read {  get; set; }

    }
}

File: Angular/MyBookShelfFrontend/src/app/Book.ts
import { Author } from "./Author";

export interface Book {
    BookId: number;
    title: string;
    description: string;
    image: File;
    authorId: number;
    author: Author;
    rating: number;
    price: string
    category: string[];
    edition: string[];
    pageNumber: string;
    alphabet: string;
    releaseDate: Date;
    youtubeLink: string;
    isbn: string
}
export enum BookEdition {
    Harcover = 0,
    Paperback = 1,
    eBook = 2,
    Audiobook = 3,
  }